### DISCUSSION TITLE
How to wrap Images in a div ?

### DISCUSSION THREAD

**Discussion by didil** [yuin/goldmark#417] [Q&A]
Hi

I'm rendering markdown docs to html and I need all rendered images to be wrapped in a div element with a specific class. Any pointers to how I can achieve that ?

Thanks !

**Comment by riscdanger** [comment]
Check out the [goldmark-figure](https://github.com/MangoUmbrella/goldmark-figure) extension. If the semantic `<figure>` tag won't work for you, you could copy the extension with [your own HTML rendering](https://github.com/MangoUmbrella/goldmark-figure/blob/3f0fbe836b84b978d6dda4f7886cfc2f88e83f19/ast/ast.go#L96-L118).
